packages/design-picker/src/types.ts

```diff
@@ -13,36 +14,40 @@ export interface Category {
 	name: string;
 }
 
+export interface DesignRecipe {
+	theme?: string;
+	patternIds?: number[];
+}
+
 export type DesignFeatures = 'anchorfm' | 'difm-lite-default'; // For additional features, = 'anchorfm' | 'feature2' | 'feature3'
 
 export interface Design {
-	categories: Array< Category >;
-	fonts?: FontPair;
-	is_alpha?: boolean;
-	is_fse?: boolean;
+	slug: string;
+	title: string;
+	recipe?: DesignRecipe;
 	is_premium: boolean;
+	categories: Category[];
+	features: DesignFeatures[];
+	is_featured_picks?: boolean; // Whether this design will be featured in the sidebar. Example: Blank Canvas
+	showFirst?: boolean; // Whether this design will appear at the top, regardless of category
```

Comment:
It's not conflicting with this sorting. It's used for blank canvas design, when `signup/design-picker-use-featured-picks-buttons` is false (i.e. the blank canvas will be the first design in every category).

But I see that the config value is `true` everywhere.

We can clean this up (and the feature flag) in a separate PR, I think.

Reply:
+1 for doing something in a seperate PR, I think that blank canvas used to be also in the "Show All" Category but it's since been taken out and maybe isn't used anymore